websites/K/Kick/presence.ts

```diff
@@ -0,0 +1,60 @@
+const presence = new Presence({ clientId: "1112901248421732462" });
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/kVabJjN.png",
+			startTimestamp: Math.floor(Date.now() / 1000),
```

Comment:
This may have the effect of setting the elapsed time to 0 all the time. Instead, store the start timestamp in a global variable.